app/code/Magento/Review/Controller/Product/ListAjax.php

```diff
@@ -22,7 +22,8 @@ class ListAjax extends ProductController implements HttpGetActionInterface
      */
     public function execute()
     {
-        if (!$this->initProduct()) {
+        if (!$this->getRequest()->isAjax() 
```

Comment:
Please format your code according to PSR-12 guidelines:
> Expressions in parentheses MAY be split across multiple lines, where each subsequent line is indented at least once. When doing so, the first condition MUST be on the next line. 
So the code would become:
```
if (
    !$this->getRequest()->isAjax() ||
    !$this->initProduct()
) {
    // code here
}
````
Reference: Section 5.1 of https://www.php-fig.org/psr/psr-12/

Reply:
Alright, I just changed with your kind indications.